frontend/src/components/cronjob/Details.tsx

```diff
@@ -93,18 +207,22 @@ export default function CronJobDetails() {
 
   const actions = [];
 
-  actions.push(
+  actions.push([
     <AuthVisible authVerb="modify" item={cronJob}>
       <ActionButton
-        description={isCronSuspended ? t('frequent|Resume') : t('frequent|Suspend')}
+        description={t('frequent|Spawn Job')}
```

Comment:
I think this `AuthVisible` component needs to check if we can create Job resources, not modify CronJob ones, right?

Reply:
updated

Reply:
This wasn't really changed to check the Job resource but I will do it.